neo/Trie/MPT/MPTProofDb.cs

```diff
@@ -0,0 +1,25 @@
+using Neo.IO;
+using Neo.Cryptography;
+using System.Collections.Generic;
+
+namespace Neo.Trie.MPT
+{
+    public class MPTProofStore : IKVReadOnlyStore
+    {
+        private Dictionary<byte[], byte[]> store = new Dictionary<byte[], byte[]>(ByteArrayEqualityComparer.Default);
```

Comment:
This doesn't need to be persistent?

Reply:
`ProofStore` is only used when verify the proofs.  Proofs, root and key are provided by users. `VerifyProof` only uses the trie logic, like rpc `invokeScript`.
I can remove it if you don't like.